src/components/main/list/ItemsTableCard.tsx

```diff
@@ -42,27 +47,25 @@ const ItemsTableCard = ({
 
   if ('files' in item) {
     return (
-      <Stack
-        direction="row"
-        gap={1}
-        boxSizing="border-box"
-        border="2px dashed grey"
-        borderRadius={2}
-        p={1}
-      >
-        <Upload /> {translateBuilder(BUILDER.UPLOAD_BETWEEN_FILES)}
-      </Stack>
+      <SmallUploadFile text={translateBuilder(BUILDER.UPLOAD_BETWEEN_FILES)} />
     );
   }
 
   return (
-    <Box px={1}>
+    <Box
+      px={1}
+      onClick={() => {
+        onClick?.(item.id);
+      }}
```

Comment:
This should work too ?

```suggestion
      onClick={() => onClick?.(item.id)}
```